spec/system/admin/vouchers_spec.rb

```diff
@@ -11,6 +11,7 @@
 
   let(:enterprise) { create(:supplier_enterprise, name: 'Feedme') }
   let(:voucher_code) { 'awesomevoucher' }
+  let(:amount) { 25 }
```

Comment:
`±` I think we also need a spec here for updating an existing voucher

Reply:
There is currently no way to update a voucher, do you still think we need a spec for it ?

Reply:
Oh, sorry I was mistaken!